src/main/java/com/github/manolo8/darkbot/modules/utils/AttackAPIImpl.java

```diff
@@ -142,7 +144,11 @@ public double modifyRadius(double radius) {
         if (!target.isMoving() || target.getHealth().hpPercent() < 0.25)
             radius = Math.min(600, radius);
 
-        return radius + heroItems.getItem(SelectableItem.Ability.ZEPHYR_MMT, ItemFlag.AVAILABLE)
+        Optional<Item> ability = heroItems.getItem(SelectableItem.Ability.ZEPHYR_MMT, ItemFlag.AVAILABLE);
+        if (ability.isEmpty())
+            ability = heroItems.getItem(SelectableItem.Ability.HECATE_PLUS_STOCKPILE, ItemFlag.AVAILABLE);
+
+        return radius + ability
                 .map(Item::getQuantity)
                 .orElse(0d) * 5;
```

Comment:
```suggestion
        return radius + heroItems.getItem(SelectableItem.Ability.ZEPHYR_MMT, ItemFlag.AVAILABLE)
            .or(() -> heroItems.getItem(SelectableItem.Ability.HECATE_PLUS_STOCKPILE, ItemFlag.AVAILABLE))
            .map(Item::getQuantity)
            .orElse(0d) * 5;
```

Java 11 has some nice optional extra operations (added in java 9), in this case, `Optional#or(Supplier<Optional>)` 
https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Optional.html#or(java.util.function.Supplier)